crates/ark/src/lsp/completions/completion_item.rs

```diff
@@ -445,29 +454,29 @@ pub(super) unsafe fn completion_item_from_symbol(
         Ok(true) => {
             // We can't even extract out the object for active bindings so they
             // are handled extremely specially.
-            return Some(completion_item_from_active_binding(name));
+            return completion_item_from_active_binding(name);
         },
         Err(err) => {
-            log::error!("Can't determine if binding is active: {err:?}");
-            return None;
+            // The only error we anticipate is the case where `envir` doesn't
+            // have a binding for `name`.
+            bail!("Failed to check if binding is active: {}", err);
```

Comment:
This is what gives us blessed silence in the logs now, when completing, e.g., `dplyr::` (or any package with re-exports).